pkg/triggers/service.go

```diff
@@ -25,34 +25,38 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-var (
-	defaultScraperInterval    = 5 * time.Second
-	defaultLeaseCheckInterval = 5 * time.Second
-	defaultMaxLeaseDuration   = 1 * time.Minute
-	defaultClusterID          = "testkube-api"
-	defaultIdentifierFormat   = "testkube-api-%s"
+const (
+	defaultScraperInterval        = 5 * time.Second
+	defaultLeaseCheckInterval     = 5 * time.Second
+	defaultMaxLeaseDuration       = 1 * time.Minute
+	defaultConditionsCheckBackoff = 1 * time.Second
+	defaultConditionsCheckTimeout = 10 * time.Second
```

Comment:
do you think 10 second is enough?

Reply:
I don't know, it seems kinda low, I'd rather put it 60-90s

Reply:
let's do 90